tests/cnf/core/network/sriov/tests/qinq.go

```diff
@@ -544,6 +578,152 @@ var _ = Describe("QinQ", Ordered, Label(tsparams.LabelQinQTestCases), ContinueOn
 		})
 	})
 
+	Context("nmstate", func() {
+		const configureNMStatePolicyName = "configurevfs"
+		var (
+			sriovInterfacesUnderTest []string
+			workerNodeList           []*nodes.Builder
+		)
+		BeforeAll(func() {
+			By("Remove non-nmstate sriov configurations")
+			removeSriovConfigurations()
```

Comment:
Why do we need it here?

Reply:
To run the nmstate with a clean env. The sriovnetworkpolicies and sriovnetworks are created in the BeforeAll before all Contexts.

Reply:
Add policy creation only to Context 802.1AD and 802.1Q and delete it from the general beforeall.
Currently it looks very confusing and massive:

```
General beforeall: Create SR-IOV configuration
"nmstate" beforeall: Remove SR-IOV configuration
```

Reply:
removed